tests/src/test/scala/com/nvidia/spark/rapids/shuffle/RapidsShuffleClientSuite.scala

```diff
@@ -98,6 +98,8 @@ class RapidsShuffleClientSuite extends RapidsShuffleTestHelper {
       val tm = ptrs(t).tableMeta
       verifyTableMeta(expected, tm)
     }
+
+    assertResult(true)(response.isClosed)
```

Comment:
```suggestion
    assert(response.isClosed)
```

Applies to other occurrences in this file as well.